roadmap/README.md

```diff
@@ -10,9 +10,81 @@ list for an overview of the initial roadmaps from the respective years.
+upon on the sideline, though with less ressources applied.
+
+### Browser Support (Substantially Upgraded)
+
+JavaScript is THE programming language for the browser. Throughout the year it became clear that we had largely underappreciated one
+central browser-related topic and challenge around our libraries: the size of the bundles created from our libraries to be possible
+to be loaded in a web context. Especially for our EVM these were several 100KBs too large to get efficiently loaded within a 
+website context. We have therefore significantly ramped up efforts here, restructured a lot of code and removed unnecessary dependencies,
+culminating in our October/November 2024 breaking releases with bundle size reductions e.g. for our EVM of 60% and greater.
+
+Together with some other browser-related improvements these releases mark the first moment in time where the Ethereum L1 protocol
+stack is now fully ready to be used within web browsers without substantial limitations.
+
+### Security (Substantially Upgraded)
+
+This topic aligns very well with the above browser support one: since the libraries are now fully browser ready (and things like
+EIP-7702 account abstraction is on the horizon) they - and particularly the EthereumJS EVM - will likely be used for more
+"serious" and security-sensitive production use cases in the future. To take this into account we gave security a lot stronger
+emphasis than initially planned. The dependency reduction from above is also a big step towards a more secure JavaScript EVM.
+Through a collaboration with Paul Millr from [noble Crypto](https://paulmillr.com/noble/) we were able to replace remaining WASM
+implementations for EVM precompiles (BLS, BN254 (aka alt_BN128)) as well as the 4844-related KZG crypto with pure JavaScript
+implementations.
+
+This makes up for a fully auditable JavaScript EVM for the first time and we plan to have an EVM audit done throughout 2025.
+
+### EIP Implementation & Testing (Expanded)
+
+The EthereumJS stack is historically being build up in a very modular way and is therefore well suited for EIP prototyping and
+early on integration. In 2024 was a year with a lot of protocol work to be done and our team increased its engagement here.
+
+Some examples are:
+
+- Pectra HF [devnet-3](https://notes.ethereum.org/@ethpandaops/pectra-devnet-3), prototyping (EIP-2935), lot of spec refinement work
+- devnet-4, previous EIP-7702 [test filling](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3737)
+- Verkle [devnets](https://github.com/ethpandaops/verkle-devnets) participation, stateless block execution, spec work
+- EIP-6493 Stable Container [Prototyping](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3452)
+
+Since autumn 2024 we are able to [fill execution-spec-tests](https://github.com/ethereum/execution-spec-tests/pull/752), which
```

Comment:
With this PR (plus the t8ntool PR on our side https://github.com/ethereumjs/ethereumjs-monorepo/pull/3603) when trying to fill fixtures (especially the 7702 ones) we also found errors in the tests themselves, thus this opens a great feedback loop with the testing team to verify the tests are correct. Also part of this feedback loop is a coverage test: for instance, when we find a new rule in 7702 which is clearly not implemented, but the test filler did not complain, this means that this rule is thus not tested in execution-spec-tests and should be added.

A side note could also be added that we have a small amount of PRs merged in EEST with new tests.

Reply:
Thanks for the summary, will put more emphasis on these kind of things in the roadmap itself (this here is only the 2024 recap, so very much just a side document).